packages/lodestar/src/chain/factory/attestation/data.ts

```diff
@@ -1,7 +1,7 @@
 import {BeaconDB} from "../../../db/api";
-import {AttestationData, BeaconBlock, BeaconState, Crosslink, Shard} from "../../../types";
+import {AttestationData, BeaconBlock, BeaconState, Crosslink, Shard} from "@chainsafe/eth2-types";
 import {getBlockRoot, getCurrentEpoch, getEpochStartSlot} from "../../stateTransition/util";
-import {FAR_FUTURE_EPOCH, GENESIS_EPOCH, ZERO_HASH} from "../../../constants";
+import {FAR_FUTURE_EPOCH, GENESIS_EPOCH, ZERO_HASH} from "../../../../../eth2-types/src/constants";
```

Comment:
i think all constats have wrong import. I saw this on a lot of places

Reply:
oh no :(